src/main/scala/sbtghactions/GenerativePlugin.scala

```diff
@@ -783,13 +783,73 @@ ${indent(jobs.map(compileJob(_, sbt)).mkString("\n\n"), 1)}
       sbt)
   }
 
-  private val readCleanContents = Def task {
-    val src = Source.fromURL(getClass.getResource("/clean.yml"))
-    try {
-      src.mkString
-    } finally {
-      src.close()
-    }
+  private def generateCleanContents(runsOnOs: String) = {
+    val first =
+      """|# This file was automatically generated by sbt-github-actions using the
+         |# githubWorkflowGenerate task. You should add and commit this file to
+         |# your git repository. It goes without saying that you shouldn't edit
+         |# this file by hand! Instead, if you wish to make changes, you should
+         |# change your sbt build configuration to revise the workflow description
+         |# to meet your needs, then regenerate this file.
+         |
+         |name: Clean
+         |
+         |on: push
+         |
+         |jobs:
+         |  delete-artifacts:
+         |    name: Delete Artifacts
+         |""".stripMargin
+
+    val last =
+      """
+        |    env:
+        |      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        |    steps:
+        |      - name: Delete artifacts
+        |        shell: bash {0}
```

Comment:
This is an addition which ensures that the script runs in bash under windows. The `{0}` part is here because grep on Windows will return an error status code if it cannot find anything causing the entire script to fail, see https://stackoverflow.com/a/73069669